### PR TITLE
[DOC]  Fix mapLayer arguments

### PR DESCRIPTION
**Pull request by genofire**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -120,8 +117,6 @@ Start a time range to put this mapLayer on first position.
 
 End a time range for first map. Stops sort this mapLayer.
 
-#### end (integer)
```
Comment (xf-) [inline README.md:123]: sry my mistake

File: README.md [hunk 2]
```diff
@@ -101,10 +101,7 @@ Interesting if one of configured map provider has zoom-level under `18`.
 
 ## mapLayers (List)
 
-A list of objects describing map layers. Each object has at least `name`
-property and optionally `url` and `config` properties. If no `url` is supplied
```
Comment (genofire) [inline README.md:105]: And optional is not any more in the text :D

**Comment by genofire** [comment]
and now?

**Comment by xf-** [comment]
https://github.com/ffrgb/meshviewer/commit/e16eb115bac60e5686c76cefbd8cf14890059825 adjusted and merged

**Pull request closed without merging**